File: OneTrack_v2.Server/DbData/Database/CompanyRequirement.cs
using System;
using System.Collections.Generic;

namespace OneTrack_v2.DbData.Models;

public partial class CompanyRequirement
{
    public int CompanyRequirementId { get; set; }

    public string WorkStateAbv { get; set; } = null!;

    public string ResStateAbv { get; set; } = null!;

    public string RequirementType { get; set; } = null!;

    public bool LicLevel1 { get; set; }

    public bool LicLevel2 { get; set; }

    public bool LicLevel3 { get; set; }

    public bool LicLevel4 { get; set; }

    public string? Document { get; set; }

    public DateTime? StartAfterDate { get; set; }
}

File: onetrack_v2.client/src/app/_Models/admin/companyRequirement.ts
export interface CompanyRequirement {
  companyRequirementId: number | null;
  workStateAbv: string | null;
  resStateAbv: string | null;
  requirementType: string | null;
  licLevel1: boolean | null;
  licLevel2: boolean | null;
  licLevel3: boolean | null;
  licLevel4: boolean | null;
  startAfterDate: string | null;
  document: string | null;
  userSOEID: string | null;
}
